ui-v2/app/styles/components/breadcrumbs.scss

```diff
@@ -1,18 +1,4 @@
+@import './breadcrumbs/index';
 main header nav:first-child {
   @extend %breadcrumbs;
 }
```

Comment:
> Is this a variation of the breadcrumbs component?

Nope this is just:

`import Breadcrumbs from 'breadcrumbs'`

or

`import .breadcrumbs from 'breadcrumbs'`

or

`import .bread-crumbs from 'breadcrumbs'`

or

`import main header nav:first-child from 'breadcrumbs'`

...aliasing


> Why doesn't it live in /breadcrumbs/skin.scss?

Because that's just the 'skin' (colors and 'decoration'), this is me using the breadcrumbs.